src/pages/DatastudioDashboard/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const Wrapper = styled.div`
+    width: 100%;
+    margin-top: -2.5rem;
+
+    @media only screen and (max-width: 768px) {
+        margin-top: -2rem;
+    }
+
+    @media only screen and (max-width: 480px) {
+        margin-top: -1rem;
+    }
+`
+
+export default function DatastudioDashboard(): JSX.Element {
+    return (
+        <Wrapper>
+            <iframe
+                title="Datastudio Dashboard"
+                width="100%"
+                height="2000px"
+                src="https://datastudio.google.com/embed/reporting/f1ce8f56-058c-4e31-bfd4-1a741482642a/page/p_miuw33ljnc"
```

Comment:
make this an env variable
have this as a default value in .env or some other idiomatic way

Reply:
Sure will refactor it a bit